tests/scripts/thread-cert/node.py

```diff
@@ -2177,14 +2177,22 @@ def get_br_on_link_prefix(self):
         self.send_command(cmd)
         return self._expect_command_output()[0]
 
-    def get_netdata_non_nat64_prefixes(self):
+    def get_netdata_non_nat64_routes(self):
         prefixes = []
```

Comment:
Suggest to rename the return variable like nat64_routes to match the renamed method.